index.js

```diff
@@ -162,39 +175,51 @@ async function terraform(args) {
     tf_fmt = status_success;
   }
 
+  if (reportDrift === true) {
+    core.startGroup('Run terraform plan');
+    const slack = new WebClient(slackBotToken);
+    const tfd = await terraform(['plan', `-lock=${terraformLock}`, `-parallelism=${terraformParallelism}`, '-out=terraform.plan', '-detailed-exitcode'].concat(terraformTargets));
+    switch (tfd.status) {
+    case 0:
+      break;
+    case 2:
+      core.warning('Terraform reported a diff');
+      const result = await slack.chat.postMessage({
+        icon_url: 'https://cdn.icon-icons.com/icons2/2107/PNG/512/file_type_terraform_icon_130125.png',
+        text: `Unapplied changes were detected for ${terraformWorkspace}:\n\`\`\`\n${tfd.stdout}\n\`\`\``,
```

Comment:
Just the workspace isn't enough, you want repo and working directory listed as well.